README.md

```diff
@@ -3,20 +3,44 @@
 This is a electrum-adapter. It exposes a Bitcoin-Core style API while using an electron API in the backend. It might be useful in specific usecases, e.g. having better performance when connecting to a electrum-server via Tor. In order to do that, it needs a Database. Quite easily you can use a kind of builtin SQLite. Depending on your usecase, you might want to use an external DB.
```

Comment:
What is the usecase that requires an external DB? These docs can be confusing, let's be specific or remove that sentence.

Reply:
Specter Cloud is using an external DB (Postgres) running Spectrum in Standalone Mode.